doc/getting_started.rst

```diff
@@ -78,6 +78,8 @@ Sometimes, you want to apply different transformations to different features:
 the :ref:`ColumnTransformer<column_transformer>` is designed for these
 use-cases.
 
+.. _pipelines:
```

Comment:
we already have a _pipeline ref (without the  s), so we might want to use another name to avoid future mismatches. Though as mentioned below I'm not sure we should touch the getting started guide anyway.